src/ui/sections/Editor/hooks/useCodeMirror.tsx

```diff
@@ -0,0 +1,90 @@
+import { defaultKeymap } from '@codemirror/commands';
+import { markdown, markdownLanguage } from '@codemirror/lang-markdown';
+import { indentOnInput } from '@codemirror/language';
+import { languages } from '@codemirror/language-data';
+import { EditorState } from '@codemirror/state';
+import { EditorView, keymap } from '@codemirror/view';
+import type React from 'react';
+import { useEffect, useRef, useState } from 'react';
+
+export const editorTheme = EditorView.theme({
+  '&': {
+    height: '75vh',
+  },
+  '.cm-content': {
+    caretColor: 'var(--cursor-color)',
+  },
+  '.cm-scroller': {
+    overflow: 'auto',
+    fontFamily: `"Roboto", "Helvetica", "Arial", sans-serif`,
+  },
+  '::-webkit-scrollbar': {
+    width: '0.5em',
+  },
+  '::-webkit-scrollbar-track': {
+    borderRadius: '20px',
```

Comment:
Let's avoid using pixel measures so that it looks the same on different monitors.

Can we switch to rem or to any other relative measure?